### ISSUE TITLE
Add Statusfy

### ISSUE THREAD

**Issue by 00willo** [awesome-selfhosted/awesome-selfhosted#1687]
Open Source Status Page system - https://github.com/bazzite/statusfy

Apache License 2.0

**Comment by worldworm** [comment]
Statusfy is a great project.
But it is already listed here: https://github.com/n1trux/awesome-sysadmin

**Comment by 00willo** [comment]
@worldworm @nodiscc  Understand, and I looked through some of the other issues raised and saw a few closed with similar already listed. When I started writing this reply, I was wondering what is the criteria to be listed on this awesome list, as I don't think it's clear.  As I hadn't yet raised a PR, I did not see the details in the pull request template, where it stated to make sure that it's not included in the awesome-sysadmin list.  Perhaps also including some details on the main `README.md` might also reduce the number of issues raised that are invalid. Possibly it could also be added to an `.github/ISSUE_TEMPLATE`.

Happy to do up a PR if you'd like.  Appreciate the effort in maintaining this list.
